Views/backend/connect/model/config/export.js

```diff
@@ -33,7 +33,10 @@ Ext.define('Shopware.apps.Connect.model.config.Export', {
 
     fields: [
         //{block name="backend/connect/model/config/import/fields"}{/block}
-        { name: 'alternateDescriptionField', type: 'string' },
+        { name: 'alternateDescriptionField', type: 'array' },
```

Comment:
Please check if it's possible to use only one data type for descriptions, string or array.